include/boundary_standard.hxx

```diff
@@ -333,4 +333,38 @@ private:
   int width;
 };
 
+/// Convert toFieldAligned
```

Comment:
Could you expand on the documentation here a little bit? These modifiers are designed to take a *function* specified in non-field-aligned/field-aligned space and apply it to a *field* specified in the other space.